src/scene/scene.js

```diff
@@ -774,6 +783,15 @@ class Scene extends EventHandler {
             this.prefilteredCubemaps = cubemaps.slice(1);
         }
     }
+
+    /**
+     * Get the lightmap pixel format.
+     *
+     * @type {number} The pixel format.
+     */
+    get lightmapPixelFormat() {
```

Comment:
this function should be moved to lightmapper, no need for a public getter like this on the scene I think

Reply:
ah actually the options builder uses it ..

Reply:
that would be ideal except standard-material-options-builder needs it.

the correct way to solve this is probably to make lightmapper's black texture the correct format (it's currently RGBA8) and then use the texture in options-builder to determine encoding.

are you ok with this change?

Reply:
I think I prefer the way it's done currently.